docs/api-overview.mdx

```diff
@@ -197,7 +203,16 @@ Let's break this down to understand what is happening here:
 }
 ```
 
-> _¹The `FullNameCredential` also has a `MiddleNameCredential`, but it's `optional`._
+
+- `PhoneCredential` is a "[single attribute atomic credential](/schema#schema-definitions)" and therefore does not have any sub-credentials. Because we specified a `allowUserInput` attribute, user will be allowed to prompt its own information for the given credential."
```

Comment:
```suggestion
- `PhoneCredential` is a "[single attribute atomic credential](/schema#schema-definitions)" and therefore does not have any sub-credentials. Because we set a `allowUserInput` attribute of `true`, the user will be given an opportunity to enter a new value if they choose."
```